src/ui/components/Icon/styles.scss

```diff
@@ -32,3 +32,29 @@
   background: url('./user-dark.svg') center no-repeat;
   background-size: contain;
 }
+
+.Icon-browse {
+  background: url('./bullet-list.svg') 50% 50% no-repeat;
+  background-size: cover;
+  height: 30px;
+  margin-right: 1em;
+  width: 30px;
+}
+
+.Icon-persona {
+  background: url('./themes.svg') 50% 50% no-repeat;
+  background-size: cover;
+  display: inline-block;
+  height: 50px;
+  margin-bottom: 15px;
```

Comment:
I don't think this bottom margin will work properly. It looks about right in English but the size of the container beside it can change based on the language. You can instead use flexbox to vertically centre it.

You'll want to set it up so you've got two containers that you can align within the parent, and fortunately it's already setup like that.

Then in CSS you can set the parent to `display: flex` and and tell the icon to be centred vertically.

```scss
.LandingPage-header {
  display: flex;
}

.LandingPage-icon {
  align-self: center;
}
```

There's a good [guide to flex box on CSS Tricks](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) that I always end up referencing since there are so many options.